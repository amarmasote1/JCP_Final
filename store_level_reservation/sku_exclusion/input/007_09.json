[
  {
     "storeId" : "ALL",
     "exclusionIndi" : "S",
     "productLine" : "804",
     "removeExclusion" : "N",
     "lotNumber" : "8041000"
  },{
     "storeId" : "ALL",
     "exclusionIndi" : "P",
     "productLine" : "804",
     "removeExclusion" : "N",
     "lotNumber" : "8041000"
  }

]
