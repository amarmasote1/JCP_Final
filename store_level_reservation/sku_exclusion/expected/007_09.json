{
	"body": [{
		"status": 212,
		"message": "Data Write Successful",
		"key": "8041000|804|ALL",
		"body": null,
		"warnings": null
	},
	{
		"status": 212,
		"message": "Data Write Successful",
		"key": "8041000|804|ALL",
		"body": null,
		"warnings": null
	}],
	"status": 221,
	"message": "Successful"
}